tensorboard/webapp/runs/views/runs_table/runs_table_test.ts

```diff
@@ -572,11 +575,86 @@ describe('runs_table', () => {
 
         const items = getOverlayMenuItems();
 
-        expect(items.map((element) => element.textContent)).toEqual([
-          'Experiments',
-          'Runs',
-          'Regex',
-        ]);
+        expect(
+          items.map((element) => element.querySelector('label')!.textContent)
+        ).toEqual(['Experiments', 'Runs', 'Regex']);
+      });
+
+      it('renders a check for a menu item that has groupBy', () => {
```

Comment:
Can we add `icon`? like `a check icon` 
I was confused and think about other check

`renders a check icon for the current groupBy menu item`

Reply:
Done. Yeah, my text does look confusing :). Thanks.